README.markdown

```diff
@@ -1,68 +1,104 @@
-# DITA Open Toolkit [![Build Status][1]](http://travis-ci.org/dita-ot/dita-ot) [![Slack][7]](http://slack.dita-ot.org/)
+# DITA Open Toolkit [![Build Status][1]](http://travis-ci.org/dita-ot/dita-ot) [![Slack][2]](http://slack.dita-ot.org/)
 
-The _DITA Open Toolkit_, or _DITA-OT_ for short, is an open-source publishing engine for XML content authored in the _Darwin Information Typing Architecture_. 
+The _DITA Open Toolkit_, or _DITA-OT_ for short, is an open-source publishing engine for XML content authored in the _Darwin Information Typing Architecture_.
 
-See [dita-ot.org][2] for documentation, information about releases, and download packages. 
+Visit the project website at [dita-ot.org][3] for documentation, information about releases, and [download packages][4].  
+For information on additional DITA and DITA-OT resources, see [SUPPORT][5].
 
-For information on additional DITA and DITA-OT resources, see [SUPPORT][8]. 
+- [Prerequisites](#prerequisites)
+- [Installing](#installing)
+- [Building output](#building-output)
+- [For developers](#for-developers)
+- [License](#license)
 
 ## Prerequisites
 
 To build and use DITA-OT, you’ll need:
 
-* Java Development Kit 8 or newer
+- Java Development Kit 8 or newer
 
-## Building
+## Installing
 
-1. Clone the DITA-OT Git repository:
+1.  Download the distribution package from [dita-ot.org/download][4].
+2.  Extract the contents of the package to the directory where you want to install the DITA-OT.
 
-        git clone git://github.com/dita-ot/dita-ot.git
+### Installing on macOS via Homebrew
 
-2. Move to the DITA-OT directory:
+On macOS, you can also install DITA-OT using the [Homebrew][6] package manager:
```

Comment:
Just a tip (that you maybe already know, but just in case):

To avoid fiddling with link reference numbers, you can just leave them out and create the links like this:

```markdown
…using the [Homebrew] package manager

[Homebrew]: https://brew..sh
```

Reply:
👍  I'm aware of that option, but I generally prefer the readability of numbered link references,  especially in cases where the same target is used for different display text, as in the download links here:  `[download packages][4]` & `[dita-ot.org/download][4]`.

I have a script that de-duplicates & re-numbers them automatically, so easy to maintain too.